### PR TITLE
Prove ? works in column wildcards

### PR DESCRIPTION
**Pull request by berland**

### CODE REVIEW

File: tests/test_ensemble.py [hunk 1]
```diff
@@ -195,6 +195,10 @@ def test_emptyens():
     assert isinstance(emptydf, pd.DataFrame)
     assert emptydf.empty
 
+    emptyparams = ens.parameters
+    assert isinstancve(emptyparams, pd.DataFrame)
```
Comment (asnyv) [inline tests/test_ensemble.py:199]:
```suggestion
    assert isinstance(emptyparams, pd.DataFrame)
```

**Review by asnyv — CHANGES_REQUESTED** [review]
typo 😄

File: tests/test_ensemble.py (hunk 1 above)
Comment (berland) [reply tests/test_ensemble.py:199]: whops,  that was not an intentional part of the PR...

**Comment by berland** [comment]
This test got added to verify a claim in: https://github.com/equinor/semeio/pull/214

**Review by asnyv — APPROVED** [review]
👍

**Pull request merged**